README.md

```diff
@@ -1,4 +1,48 @@
-yolo-hipster
-============
+taxster
+=======
 
-Naive Bayes taxonomy classifier using scikit-learn
+This library contains tools for working with taxonomic assignment data. This should be considered pre-alpha software. **Public APIs can and will change!**
+
+Install
+-------
+
+pip install https://github.com/biocore/taxster/archive/master.zip
+
+Consensus taxonomy assignments from .uc files
+---------------------------------------------
+
+This example illustrates how to apply the consensus taxonomy processing described in [Bokulich et al (in preparation)](https://peerj.com/preprints/934/). It assumes that you have the test data files included in this repository in ``./test-data``.
+
+```python
+import taxster
+
+taxonomy_map = {}
+for line in open('./test-data/uc/tax-map.tsv', 'U'):
```

Comment:
It's just a code example in the README so do you guys mind if I just leave as-is for now? I need to work on something else, and I'd like to just get it in.